src/routes/search/containers/SearchModalContainer.js

```diff
@@ -15,7 +15,7 @@ type OwnPropsType = {| navigation: NavigationScreenProp<*> |}
 
 export type PropsType = {|
   categories: CategoriesMapModel | null,
-  navigateToCategory: NavigateToCategoryParamsType => StoreActionType,
+  navigateToCategory: NavigateToCategoryParamsType => void,
```

Comment:
I wrote earlier that you should return the result of the dispatch. I think this is the type you changed to match this.

Not sure if this return type makes sense or not. Maybe void is better.

Reply:
I think it's not really important if we return the action or not, as long as its's consistent. Imo it looks a bit cleaner without the returned action, since it's just a general `StoreActionType` anyway. If we want return types, it should be the actual `ActionType`.

Reply:
Alright, @maxammann and I think both it's good as it is.